packages/mdc-tab-scroller/README.md

```diff
@@ -65,6 +65,9 @@ CSS Class | Description
 `mdc-tab-scroller` | Mandatory. Contains the tab scroller content.
 `mdc-tab-scroller__scroll-area` | Mandatory. Denotes the scrolling area.
 `mdc-tab-scroller__scroll-content` | Mandatory. Denotes the scrolling content.
+`mdc-tab-scroller--start-aligned` | Optional. Sets the elements inside the scroll content element to be aligned to the start of the scroll content element.
```

Comment:
Nit: `--align-start`, `--align-end`, and `--align-center` would...align...with modifier classes we have on a few other components